### DISCUSSION TITLE
Easiest Way (without loading catalog) to Go to Location on the Map

### DISCUSSION THREAD

**Discussion by tech-maven-geospatial** [TerriaJS/terriajs#4893] [Q&A] [answered]
We've build a Go to Coordinate function (MGRS, GARS, PlaceKey, PlusCode, What3Words and Projected Lat Long - get converted to WGS84 Decimal Degree Latitude and Longitude)
Having trouble moving map to that location.  
Someone on gitter suggested configuring this as a search provider

This new feature will be found in the Mobile and Desktop version of Earth Explorer 3D Map app
https://apps.apple.com/np/app/earth-explorer-3d-map-app/id1485111764
https://play.google.com/store/apps/details?id=com.app.earthexplorer&hl=en_US

**Comment by KeyboardSounds** [comment, marked as answer]
It's hard for me to answer this without having a better idea what sort of workflow you're looking for. Would you be able to describe the steps that you want a user to carry out to make this happen?

A search provider would be one approach. Have a look at the SearchProvider classes in here: https://github.com/TerriaJS/terriajs/tree/master/lib/ViewModels

Another way is to add a button somewhere that does something like [this code](https://github.com/TerriaJS/terriajs/blob/0aac25567268b6856b3597a6112f9c5cf88fdd0c/lib/ViewModels/GazetteerSearchProviderViewModel.js#L215).
